### ISSUE TITLE
auto-fetch the newest version of ghost

### ISSUE THREAD

**Issue by simonv3** [jparyani/GhostSS#12]
I noticed in Oasis that ghost is v0.5.8 while the most recent version is 0.6.4

I'm not sure whether this is possible within sandstorm, but what I've seen done for a webfaction install script of ghost is to just install ghost as an npm package (example here: https://github.com/mozz100/ghost-on-webfaction) and this way when it gets installed it's just a matter of running npm update. 

But like I said, I'm not sure whether that makes sense here. It might be better to just replace everything with the most recent version.

**Comment by ocdtrekkie** [comment]
There is probably a lot of chance of breaking the Sandstorm-specific modifications in just auto-pulling code from upstream.

**Comment by jparyani** [comment]
Yep, unfortunately most of the work in updating to the latest version is handling the rebase/merge conflicts. It's nearly impossible to get working in an automated manner unless Sandstorm's changes got upstreamed.

Becoming upstreamed is an eventual goal, so I'll leave this issue open to track it.

**Comment by html5cat** [comment]
Happy new year! Many folks probably want to get into blogging (myself including) so it would be great to have ghost up to date. Pretty please? 😅

**Comment by compuguy** [comment]
@html5cat I'm wondering the same thing. The current upstream build is 0.11.0, so this fork is very, very old now...

**Comment by jparyani** [comment]
The main blocker on updating at the moment is that the auth code for Ghost has completely changed. I believe they've arrived on a final version, so once I (or someone else) fixes the Sandstorm port, an autoupdate mechanism might become more feasible.
